docs/index.rst

```diff
@@ -595,6 +678,50 @@ Users
        "last_name": "Watson"
      }
 
+.. http:post:: /password-resets/
```

Comment:
I'm not unhappy with the current approach. Maybe `confirmations` is a bit to generic though. Would `password-confirmations` or `password-changes` maybe be better (to go with `password-resets`)?